### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by ManuhuiaBarcham**
Update our main branch with the work we did on the controller section tonight (08/18/2021)

### CODE REVIEW

**Review by michellecoder — CHANGES_REQUESTED** [review]
For the controllers api it should be tutor not teacher.

**Review by MohamedMesahel — APPROVED** [review]

**Review by MohamedMesahel — APPROVED** [review]

File: controllers/api/formsRoutes.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    });
+//     app.get('/newTeacher', function(req, res) {
+//         res.render('newTeacher');
+//     });
 
-    app.get('/newStudent/:pid', function(req, res) {
-        res.render('newStudent',{teacherId:req.params.pid});
-    });
+//     app.get('/newStudent/:pid', function(req, res) {
+//         res.render('newStudent',{teacherId:req.params.pid});
```
Comment (MaryVPie) [inline controllers/api/formsRoutes.js:8]: Here is teacherID. Please change it into tutorId

File: controllers/api/formsRoutes.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+//     });
 
-    app.get('/scheduleStudent', function(req, res) {
-        res.render('scheduleStudent');
-    });
-}
+//     app.get('/scheduleStudent', function(req, res) {
```
Comment (MaryVPie) [inline controllers/api/formsRoutes.js:11]: Do we still need schedule?

File: controllers/api/index.js [hunk 3]
```diff
@@ -1,15 +1,8 @@
...
+const tutorRoutes = require('./tutorRoutes');
 
-
-router.use('/users', userRoutes);
-router.use('/role', roleRoutes);
-router.use('/language', languageRoutes);
-router.use('/userlanguage', userlanguageRoutes);
-router.use('/forms', formsRoutes )
+router.use('/students', studentsRoutes);
+router.use('/tutors', tutorRoutes);
```
Comment (MaryVPie) [inline controllers/api/index.js:7]: Is it a typo "s" in studentSRoutes and in /students, /tutors?

File: controllers/api/roleRoutes.js [hunk 4]
```diff
@@ -1,83 +1,83 @@
...
-    }).then(function(dbStudent) {
-      JSON.stringify(dbStudent);
-      res.json(dbStudent);
-    });
-  });
+// module.exports = function(app) {
+//   //get all student
+//   app.get('/api/student', function(req, res) {
+//     db.Student.findAll({
+//       include: [db.Teacher, db.Schedule]
```
Comment (MaryVPie) [inline controllers/api/roleRoutes.js:7]: db.Teacher -> db.Tutuor

File: controllers/api/roleRoutes.js [hunk 5]
```diff
@@ -1,83 +1,83 @@
...
-    }).then(function(dbStudent) {
-      JSON.stringify(dbStudent);
-      res.json(dbStudent);
-    });
-  });
+//   //get Student by id
+//   app.get('/api/student/:id', function(req, res) {
+//     db.Student.findAll({
+//       where: {id: req.params.id},
+//       include: [db.Teacher, db.Schedule]
```
Comment (MaryVPie) [inline controllers/api/roleRoutes.js:18]: db.Teacher -> db.Tutuor

File: controllers/api/roleRoutes.js [hunk 6]
```diff
@@ -1,83 +1,83 @@
...
-    });
-  });
+//   //Add new Student
+//   app.post('/api/student', function(req, res) {
+//     //console.log(req.body);
+//     db.Student.create(req.body).then(function (dbStudent) {
+//       console.log(dbStudent);
+//       res.render('schedulestudent', {
+//         studentid: dbStudent.id,
+//         pid: req.body.TeacherId
```
Comment (MaryVPie) [inline controllers/api/roleRoutes.js:37]: Teacher -> Tutuor

File: controllers/api/roleRoutes.js [hunk 7]
```diff
@@ -1,83 +1,83 @@
...
-      where: {
-        id: req.params.id
-      }
-    }).then(function (dbStudent) {
-      res.json(dbStudent);
-    });
-  });
-};
+//   app.delete('/api/child/:id', function(req, res) {
+//     db.Student.destroy({
```
Comment (MaryVPie) [inline controllers/api/roleRoutes.js:75]: What is child here?

File: controllers/homeRoutes.js [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+const withAuth = require('../utils/auth');
+
+router.get('/', async (req, res) => {
+  try {
+    // Get all projects and JOIN with user data
+    const projectData = await Project.findAll({
+      include: [
+        {
+          model: User,
+          attributes: ['name'],
```
Comment (MaryVPie) [inline controllers/homeRoutes.js:12]: Do we use first name and last name?

**Review by MaryVPie — COMMENTED** [review]
Please have a look at my comments

**Review by michellecoder — APPROVED** [review]

**Pull request merged**